src/main/java/gov/cms/madie/models/measure/Measure.java

```diff
@@ -112,6 +112,7 @@ public class Measure extends ResourceAcl {
   @Singular("rav")
   private List<DefDescPair> riskAdjustments;
   private ProgramUseContext programUseContext;
+  private String measureBundleJson;
```

Comment:
as mentioned in Slack, should we save this directly to the measure? As it will be a large field, we may want to store this in a separate collection and only fetch when actually needed for an operation.